src/app/(marketing)/profile/[username]/constants.ts

```diff
@@ -0,0 +1,14 @@
+export const MAX_LEVEL = 100
+export const RANKS = [
+  { color: "#78c93d", title: "novice" },
+  { color: "#88d525", title: "learner" },
+  { color: "#f6ca45", title: "knight" },
+  { color: "#78c93d", title: "veteran" },
+  { color: "#ec5a53", title: "champion" },
+  { color: "#53adf0", title: "hero" },
+  { color: "#c385f8", title: "master" },
+  { color: "#3e6fc3", title: "grand master" },
+  { color: "#be4681", title: "legend" },
+  { color: "#000000", title: "mythic" },
+  { color: "linear-gradient(to left top, #00cbfa, #b9f2ff)", title: "???" },
```

Comment:
??? :D

Reply:
Don't judge me. I'm just doing what I have to do. :laughing: